UKHomeOffice/sgmr-service
# Rename and restructure files for more clarity

To test this was successful

- go to _vessels_
- go to _save_a_vessel_ : form should load, save, and error as it did previously (note: validation has not yet been updated with the new rules)
- go to _vesselName_ to edit a vessel : form should load, save, and error as it did previously (this includes the new validation rules)
